src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -425,44 +432,62 @@ Environment variable: `$WEB_EXT_API_SECRET`
+
+::: note alert
+Proxies are not currently supported with the add-on submission API. See https://github.com/mozilla/web-ext/issues/2472.
+:::
 
 Environment variable: `$WEB_EXT_API_PROXY`
 
 <section id="channel"></section>
 
 #### `--channel`
 
-This specifies the `channel` in which the extension is signed. It defaults to `unlisted` or the `channel` of your extension's latest version. The values for `channel` are:
+This specifies the `channel` in which the extension is signed. For the signing API it defaults to `unlisted` or the `channel` of your extension's latest version. For the add-on signing API the variable is required and has no default. The values for `channel` are:
 
 <div class="table-wrapper table-scroll">
 
 | Channel    | Result                                                                                                                                                                                                              |
 | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `listed`   | The extension gets submitted for public listing on [addons.mozilla.org](https://addons.mozilla.org). This type of channel is not well supported and cannot be used for some cases, as documented below. |
+| `listed`   | The extension gets submitted for public listing on [addons.mozilla.org](https://addons.mozilla.org). With the signing API this type of channel is not well supported and cannot be used for some cases, as documented below. (The add-on submission API has full support.) |
 | `unlisted` | The extension gets submitted for signing for the purpose of [self-distribution](/documentation/publish/self-distribution/) on your own website.                                                                                                                                                      |
 
 </div>
 
 One example of using the `--channel` option is to [create a beta version](/documentation/develop/getting-started-with-web-ext/#signing-test-version-listed) for a `listed` extension (that is, one you have already [submitted to addons.mozilla.org](/documentation/publish/submitting-an-add-on/)).
 
 ::: note alert
-Setting `--channel=listed` for a new extension is not yet supported. See [https://github.com/mozilla/web-ext/issues/804](https://github.com/mozilla/web-ext/issues/804)
+With the signing API, setting `--channel=listed` for a new extension is not yet supported. See [https://github.com/mozilla/web-ext/issues/804](https://github.com/mozilla/web-ext/issues/804). (The add-on submission API has full support.)
```

Comment:
Yeah, it makes the `channel` section in particular a real mess but I wasn't sure how to structure it because all of the notes and warnings only apply to the old signing api.

Reply:
so should we revert these changes and the other ones below?

Reply:
```suggestion
Setting `--channel=listed` for a new extension is not yet supported. See [https://github.com/mozilla/web-ext/issues/804](https://github.com/mozilla/web-ext/issues/804)
```